core/src/main/resources/hudson/model/Job/permalinks_pl.properties

```diff
@@ -20,4 +20,4 @@
 # OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN
 # THE SOFTWARE.
 
-Permalinks=Odno\u015Bniki permanentne
+Permalinks=Odno\u015Bniki
```

Comment:
Actually I tend to disagree. There's a semantic difference between a link
and permalink.
The second's supposed to convey a notion of permanentness. I.e somewhat
'guaranteed' to be stable over time.

In french for example, permalien is a neologism too. But it's now somehow
used in many places.

Or maybe Polish already that permanence characteristics in the 'link'
equivalent?
Le 2 févr. 2016 7:33 PM, "Damian Szczepanik" <notifications@github.com> a
écrit :

> In core/src/main/resources/hudson/model/Job/permalinks_pl.properties
> <https://github.com/jenkinsci/jenkins/pull/2008#discussion_r51612341>:
>
> > @@ -20,4 +20,4 @@
> >  # OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN
> >  # THE SOFTWARE.
> >
> > -Permalinks=Odno\u015Bniki permanentne
> > +Permalinks=Odno\u015Bniki
>
> This is because we don't put permanent as adjective for links and there is
> no need to make distinction.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/jenkinsci/jenkins/pull/2008/files#r51612341>.
>